java8/src/main/java/cucumber/runtime/java8/Java8HookDefinition.java

```diff
@@ -1,17 +1,17 @@
-package cucumber.runtime.java;
+package cucumber.runtime.java8;
+
+import static java.util.Arrays.asList;
```

Comment:
Looks like I have the ability to quickly format the imports of the entire project. We can pick a standard now and adhere to it.

https://www.jetbrains.com/help/idea/optimizing-imports.html